scripts/static-styles/index.js

```diff
@@ -15,6 +15,7 @@
 
 import fs from 'fs';
 import path from 'path';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
added this line because eslint wasn't allowing me to commit changes -
```bash
18:1  error  'prettier' should be listed in the project's dependencies. Run 'npm i -S prettier' to add it  import/no-extraneous-dependencies
```